app.js

```diff
@@ -0,0 +1,44 @@
+const express = require("express");
+const cors = require("cors");
+const bodyParser = require("body-parser");
+const app = express();
+const axios = require("axios")
+
+const port = 3000;
+
+let animals = ["sealion", "lion", "tiger"]
+
+app.use(cors());
+
+//  ANIMAL APP
+// const isAnimal = (req, res, next) => {
+//     let result = req.params
+//     if(animals.includes(result['animal'])){
+//         res.json({status: 200, message: true})
+//     } else {
+//         res.json({status: 200, message: false})
+//     }
+//     next()
+// }
+
+// app.get("/animals/:animal", isAnimal)
+
+// RANDOM NUMBERS APP
+const generateSpread = (req, res, next) => {
+    let n1 = req.params["num1"]
```

Comment:
turn the params into integers here, this would make your res.json look a lot cleaner